bvm/ballerina-rt/build.gradle

```diff
@@ -84,6 +84,7 @@ dependencies {
     dist 'org.slf4j:slf4j-api:1.7.22'
     dist 'com.moandjiezana.toml:toml4j:0.7.2'
     dist 'org.apache.commons:commons-lang3:3.8.1'
+    dist 'org.apache.commons:commons-text:1.9'
```

Comment:
Are we going to remove this.

Reply:
toml-parser depends on this package.  So we can't remove this